components/src/core/Drawer/DrawerNavGroup.tsx

```diff
@@ -162,6 +162,10 @@ export const DrawerNavGroup: React.FC<DrawerNavGroupProps> = (props) => {
     const defaultStyles = makeStyles(props, theme, insets, fontScale);
     const { activeItem } = useDrawerContext();
 
+    const defaultProps: Partial<DrawerNavGroupProps> = {
+        hidePadding: true,
```

Comment:
`hidePadding` does not have a default value in the DrawerNavGroup. `titleDivider`, `items`, `styles` do